src/styles/Book.css

```diff
@@ -1,4 +1,225 @@
 .Book {
+  display: grid;
+  grid-template-columns: 3fr 2fr 2fr;
+  border: 1px solid #e8e8e8;
+  height: 10.625rem;
+  width: 85%;
+  border-radius: 4px;
+  margin: 0.487rem 0;
+  background-color: #fff;
+}
```

Comment:
- [OPTIONAL]  Books are looking good! Why don't you try with `flexbox` for your container? This way it would be easier to make your app responsive as this is a good project to show in your portfolio.